pom.xml

```diff
@@ -16,20 +16,42 @@
   </parent>
 
   <properties>
-    <lombok.version>1.16.20</lombok.version>
+    <lombok.version>1.18.16</lombok.version>
+    <hibernate-validator.version>6.0.10.Final</hibernate-validator.version>
   </properties>
 
   <dependencies>
     <dependency>
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter-data-jpa</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-security</artifactId>
+    </dependency>
     <dependency>
       <groupId>org.projectlombok</groupId>
       <artifactId>lombok</artifactId>
       <version>${lombok.version}</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.hibernate</groupId>
+      <artifactId>hibernate-validator</artifactId>
+      <version>${hibernate-validator.version}</version>
+    </dependency>
```

Comment:
if DTO is moved to backend, then remove jackson

Reply:
removed